addon/services/engine-router-service.js

```diff
@@ -37,16 +37,24 @@ export default class EngineRouterService extends Service.extend(Evented) {
     this.trigger('routeDidChange', ...args);
   }
 
-  @reads('externalRouter.rootURL') rootURL;
+  @dependentKeyCompat
```

Comment:
this would allow property to be used in computed properties, this is more for backward compatibility rather than any new feature

Reply:
Can we use `getters` here? or computed?

Reply:
@villander this is actually turned into a getter, `@dependentKeyCompat` is only needed if someone decide to use this property within the computed property.
Actually, after second though, let's keep it `computed` for the time being